asb-native/src/main/java/org/ballerinax/asb/sender/MessageSender.java

```diff
@@ -195,7 +269,7 @@ public Object sendBatch(BMap<BString, Object> messages) {
     public Object closeSender() {
         try {
             sender.close();
-            log.debug("Closed the sender");
+            log.debug("Closed the sender. Idetifier=" + sender.getIdentifier());
```

Comment:
Shall we use a string template here?
```suggestion
            log.debug(strnig `Closed the sender. Idetifier = ${sender.getIdentifier()}`);
```

Reply:
This is a java file. We can use java string formatting with [1] or stringBuilder, but as string is relatively simple, IMO we do not need to go that level. Later JDK versions would get more simpler way to do this [2]. 

[1]. https://www.baeldung.com/java-string-formatter
[2]. https://bugs.openjdk.org/browse/JDK-8273943